lib/datadog/tracing/contrib/pg/instrumentation.rb

```diff
@@ -0,0 +1,75 @@
+# typed: false
+
+require 'datadog/tracing'
+require 'datadog/tracing/metadata/ext'
+require 'datadog/tracing/contrib/analytics'
+require 'datadog/tracing/contrib/pg/ext'
+
+module Datadog
+  module Tracing
+    module Contrib
+      module Pg
+        # PG::Connection patch module
+        module Instrumentation
+          def self.included(base)
+            base.prepend(InstanceMethods)
+          end
```

Comment:
@delner I've seen this pattern before and I'm not sure why we do this (include to then prepend).
Do you know of any reason to not directly prepend our instrumentation module?

Reply:
It necessary when patching a blend of class and instance methods. As a matter of convenience, it also removes the ambiguity of when to `include` and when to `prepend` by encapsulating the correct behavior in the module. Moreover, it means you can also apply multiple modules in one `include` call. See `Workers::Polling` as an example.

Here, it's not strictly necessary, and I'm agnostic to whether we repeat the pattern or not.